components/common/src/command/package/install.rs

```diff
@@ -852,11 +852,9 @@ impl<'a> InstallTask<'a> {
         channel: &Channel,
         token: Option<&str>,
     ) -> Result<FullyQualifiedPackageIdent> {
-        let origin_package: PackageIdent = self
-            .depot_client
-            .show_package(ident, Some(channel.0), token, None)?
-            .ident
-            .into();
+        let origin_package = self
+            .api_client
+            .show_package(ident, Some(channel.0), token, None)?;
```

Comment:
This is my favorite payoff of not leaking that deserialize-only internal struct: code deletion!